packages/amplify_datastore/example/integration_test/query_test/standard_query_operations_test.dart

```diff
@@ -14,23 +14,23 @@
  */
 
 import 'package:amplify_datastore/amplify_datastore.dart';
+
 import 'package:amplify_datastore_example/models/ModelProvider.dart';
 import 'package:integration_test/integration_test.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:amplify_flutter/amplify.dart';
 
-import 'utils/setup_utils.dart';
+import '../utils/setup_utils.dart';
 
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
-  group('query', () {
+  group('standard query operations', () {
```

Comment:
nit: What does the "standard" refer to? 😅

Reply:
I couldn't think of a good name for this group of tests. Its basically the query operations that don't have any arguments (queryPredicate, sort) and aren't focused on testing a specific data type. Maybe "basic query functionality" would be better?